src/ieee80211_station.erl

```diff
@@ -1733,3 +1733,48 @@ handle_session_timer_ev({_, Level, _} = Ev, {Interval, _, _Opts} = Timer,
 		Data0
 	end,
     {keep_state, Data}.
+
+add_location(AC, Map) when is_binary(AC) ->
```

Comment:
This and next function causes are too similar, it make sense create a small helper function like to_bin/1 which you can use in one place and not repeat the code

Reply:
Refactored.